rabbitmq.nuspec

```diff
@@ -14,13 +14,8 @@ The RabbitMQ Chocolatey package source has a new home at https://github.com/tnwi
 #### Package Parameters
 
  * `/NOMANAGEMENT` - this setting will override the default enabling of the RabbitMQ management console'
- * `/RABBITMQBASE` - specify an optional RABBITMQ_BASE. Note the parameter has no underscore, but the environment variable does. ex:/RABBITMQBASE:C:\ProgramData\RabbitMQ. The default is %AppData\RabbitMQ'
-
-3.5.3: Un-installation is now silent, no user interaction is required.

```

Comment:
I think this could go in the release notes if wanted, but I don't see the point. As long as the package works, version history seems irrelevant to the package consumers.